examples/set_policy.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+#
+# Copyright 2019 Venafi, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from pprint import pprint
+
+from parser import json_parser, yaml_parser
+from parser.utils import parse_policy_spec
+from policy.policy_spec import PolicySpecification, Policy, Subject, KeyPair, SubjectAltNames, Defaults, DefaultSubject, \
+    DefaultKeyPair
+from vcert import venafi_connection
+import logging
+from os import environ
+
+logging.basicConfig(level=logging.INFO)
+logging.getLogger("urllib3").setLevel(logging.ERROR)
+
+
+def main():
+    # Get credentials from environment variables
+    zone = environ.get('CLOUD_ZONE')
+    api_key = environ.get('CLOUD_APIKEY')
+
+    # Get connector object
+    conn = venafi_connection(api_key=api_key)
+
+    # Define policy specification object to create a new policy
+    ps = PolicySpecification()
+    # Alternatively, the parser utilities can be used to read a json/yaml file into a PolicySpecification object
+    # ps = json_parser.parse_file('path/to/file.json')
+    # ps = yaml_parser.parse_file('path/to/file.yaml')
+
+    # All of the following values can be omitted to create a Policy with inherited (TPP) or recommended (Cloud) settings
+    ps.users = ["Ryan", 'Amy']
+    ps.approvers = ["Tom", "Mary"]
+    ps.owners = ['venafi']
+    ps.user_access = 'some data here'
```

Comment:
I think it is best to leave the `users`, `approvers`, `owners`, and `user_access` out of the example because they currently don't do anything

Reply:
done